.travis.yml

```diff
@@ -1,5 +1,18 @@
 language: rust
+
+cache: cargo
+
 rust:
   - stable
   - beta
   - nightly
+    
+os:
+  - linux
+
+script:
+  - cargo build --verbose
+  - cargo test --verbose
+
+notifications:
+  email: false
```

Comment:
I don't want to seem pedantic, but there's a newline at the end of file missing :)